osu.Game/Overlays/DirectOverlay.cs

```diff
@@ -313,16 +298,14 @@ private void updateSearch()
             api.Queue(getSetsRequest);
         }
 
+        private int distinctCount(List<string> list) => list.Distinct().ToArray().Length;
+
         protected override void PopOut()
         {
+            previewTrackManager.CurrentTrack?.Stop();
```

Comment:
i am not sure globally stopping the current track on every overlay is the correct way forward. the old logic used to be aware of whether the track playing was owned by one of our children, while this is no longer the case.

Reply:
Looks like my mistake. This can be resolved by adding `PreviewTrack.Owner` property of type `OverlayContainer`.